exponax/nonlin_fun/_convection.py

```diff
@@ -37,19 +39,41 @@ def __init__(
         channels as spatial dimensions and then gives
 
         ```
-            𝒩(u) = - b₁ 1/2 ∇ ⋅ (u ⊗ u)
+            𝒩(u) = -b₁ u ⋅ ∇ u
         ```
-
+        
         with `∇ ⋅` the divergence operator and the outer product `u ⊗ u`.
+
+        Meanwhile, if you use a conservative form, the convection term is given by
+
+        ```
+            𝒩(u) = -b₁ u (u)ₓ
+        ```
+        
+        for 1D and
+
+        ```
+            𝒩(u) = -b₁ ∇ ⋅ (u ⊗ u)
+        ```
+
+        for 2D and 3D.
+
         Another option is a "single-channel" hack requiring only one channel no
         matter the spatial dimensions. This reads
 
         ```
-            𝒩(u) = - b₁ 1/2 (1⃗ ⋅ ∇)(u²)
+            𝒩(u) = -b₁ 1/2 (1⃗ ⋅ ∇)(u²)
+        ```
+        
+        for the conservative form and
+        
+        ```
+            𝒩(u) = -b₁ 1/2 u (1⃗ ⋅ ∇)u
```

Comment:
non-conservative single-channel should not have the 1/2, right? (cf table in https://github.com/Ceyron/exponax/issues/43#issuecomment-2421861564)